CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unpublished]: https://github.com/file-icons/atom/compare/v2.1.47...HEAD
 
+[v2.1.48] - 2021-03-13
```

Comment:
```suggestion
[Unpublished]
```

The timestamped version-string headers are only for releases that've been published.

Reply:
done